### ISSUE TITLE
Downloads don't work since the update

### ISSUE THREAD

**Issue by tonjoi** [subhra74/xdm#66]
I've been a user of XDM for over a year now.
Last week I was prompted to update from version 2018 to 2020. But since I did it all downloads stop working properly. Youtube downloads are empty (no sound or video) in spite of the format.

My OS is Win10_64 with a chrome browser.

**Comment by subhra74** [comment]
Did you update the chrome extension?
Remove the old extension and install the new one from XDM's browser monitoring page

**Comment by ghalibxlib** [comment]
Hi @subhra74 , I can't download .rar file since the update. Every time I try to download it the file extension change into .html

**Comment by RomjanHossain** [comment]
@ghalibxlib, I have this problem on some websites too. When I use it on Chrome browser to download files it changes to .html file. but when i'm doing it with firefox it works fine.

**Comment by subhra74** [comment]
Yes, this is currently an issue with chrome add-on, which will be fixed soon

**Comment by DBDigital** [comment]
I'm also running into this after updating to 7.2.10 from  2018-09-23 (7.2.7).  Downloads won't actually start.  I'm using Firefox though.  Uninstalled, reinstalled same problem.  Reverted back to 7.2.7 and it works fine again.  Doesn't seem to matter if I add a URL directly or not either.

7.2.7 works great.  The only thing I have found is the bandwidth limiter seems to be off.  Set it for 200 and it does 300-350.  Set it for 100 and it downloads at 200 (or about).  Not a big deal as I can just compensate by setting it lower.

**Comment by subhra74** [comment]
On firefox it should work, What happens when download does not start. Does the firefox default downloader starts the download? Are you using any VPN or similar application on browser.

**Comment by DBDigital** [comment]
It hands off to XMD, I get the popup and I say where to save the file, but where before it started showing the download speed and tracking progress, it just sits there at 0.   Everything works as before with the previous version except it doesn't actually download.  I'm not using a VPN or anything similar.
